src/js/main.js

```diff
@@ -1 +1,8 @@
-console.log('Write your hw 3 code here');
+const form = document.forms[0];
+const payButton = form.elements.submit;
+
+payButton.addEventListener('click', function(e) {
+  e.preventDefault();
+  form.classList.toggle('modal--full');
```

Comment:
`modal--full` is the right class...but is `form` the right recipient for this class? 🤔

I think this GIF should help:

![checkout-classing](https://res.cloudinary.com/yicf/image/upload/w_500/v1561977776/Code%20The%20Web/checkout-classing.gif)

Reply:
TIP — Before jumping into actual coding, it's typically helpful to get a visual what a class is doing through the dev tools:

![visual](https://res.cloudinary.com/yicf/image/upload/w_400/v1562687846/Code%20The%20Web/add-class.jpg)